src/status_im/multiaccounts/login/core.cljs

```diff
@@ -128,6 +129,16 @@
    current-network
    (types/deserialize networks)])
 
+(defn convert-multiaccounts-addresses
+  [multiaccount]
+  (-> multiaccount
```

Comment:
Nitpicking: maybe it's nicer to let-bind update-fn:
```
(let [update-fn #(update % :address eip55/address->checksum)]
    (-> multiaccount
        update-fn
        (update :accounts #(mapv update-fn %))))
```

Reply:
done